### PR TITLE
initial packaging for Aravis library (industrial cameras) and related utils

### PR DESCRIPTION
**Pull request by eudoxos**

### CODE REVIEW

File: mingw-w64-aravis/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Maintainer: Václav Šmilauer <eu@doxos.eu>
+
+_realname=aravis
+_REALNAME=ARAVIS
```
Comment (lazka) [inline mingw-w64-aravis/PKGBUILD:4]: remove

**Comment by lazka** [comment]
lgtm

File: mingw-w64-aravis/PKGBUILD (hunk 1 above)
Comment (eudoxos) [reply mingw-w64-aravis/PKGBUILD:4]: Being new to msys2, I need perhaps more explanation. I used another package (can't remember which one) and most of them use `_realname` and have the maintainer comment. When I remove `_realname`, I will just write `aravis` verbatim everywhere where it appears, that's the way it whould be? Thanks.

Comment (lazka) [reply mingw-w64-aravis/PKGBUILD:4]: I meant the uppercase `_REALNAME`. Why do you think it is needed?

Comment (eudoxos) [reply mingw-w64-aravis/PKGBUILD]: okay, a leftover from the package I started from. deleted.

**Pull request merged**